scipy/stats/_distribution_infrastructure.py

```diff
@@ -3548,14 +3558,16 @@ def make_distribution(dist):
         ``moment``, and ``sample``.
         If defined, these methods must accept the parameters of the distribution as
         keyword arguments and also accept any positional-only arguments accepted by
-        the corresponding method of `ContinuousDistribution`. The ``moment`` method
-        must accept the ``order`` and ``kind`` arguments by position or keyword, but
-        may return ``None`` if a formula is not available for the arguments; in this
-        case, the infrastructure will fall back to a default implementation. The
-        ``sample`` method must accept ``shape`` by position or keyword, but contrary
-        to the public method of the same name, the argument it receives will be the
-        *full* shape of the output array - that is, the shape passed to the public
-        method prepended to the broadcasted shape of random variable parameters.
+        the corresponding method of `ContinuousDistribution`. When multiple
+        parameterizations are defined, these methods must accept all parameters from
+        all parameterizations. The ``moment`` method must accept the ``order`` and
+        ``kind`` arguments by position or keyword, but may return ``None`` if a
+        formula is not available for the arguments; in this case, the infrastructure
+        will fall back to a default implementation. The ``sample`` method must accept
+        ``shape`` by position or keyword, but contrary to the public method of the
+        same name, the argument it receives will be the *full* shape of the output
+        array - that is, the shape passed to the public method prepended to the
+        broadcasted shape of random variable parameters.
```

Comment:
Just want to see if anything else changed. GH is not showing the diff very nicely.
```suggestion
        the corresponding method of `ContinuousDistribution`. 
        When multiple parameterizations are defined, these methods must accept
        all parameters from all parameterizations. The ``moment`` method
        must accept the ``order`` and ``kind`` arguments by position or keyword, but
        may return ``None`` if a formula is not available for the arguments; in this
        case, the infrastructure will fall back to a default implementation. The
        ``sample`` method must accept ``shape`` by position or keyword, but contrary
        to the public method of the same name, the argument it receives will be the
        *full* shape of the output array - that is, the shape passed to the public
        method prepended to the broadcasted shape of random variable parameters.
```